content/pt-br/docs/reference/glossary/app-container.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Contêiner de aplicações
+id: app-container
+data: 2023-01-24
+full_link:
+short_description: >
+   Um contêiner usado para executar parte de uma carga de trabalho. Pode ser comparado com um contêiner de inicialização (init container).
+
+aka:
+tags:
+- workload
+---
+  Contêineres de aplicações (ou app containers) são os {{< glossary_tooltip text="contêineres" term_id="container" >}} de um {{< glossary_tooltip text="Pod" term_id="pod" >}} que são inicializados depois que os {{< glossary_tooltip text="contêineres de inicialização (init container)" term_id="init-container" >}} tenham sido concluídos.
+
+<!--more-->
+
+Um contêiner de inicialização (init container) permite separar os detalhes de inicialização que são importantes para a
+{{< glossary_tooltip text="carga de trabalho" term_id="workload" >}}, e que não precisam continuar em execução
+assim que o contêiner de aplicação for iniciado.
+Se um Pod não tiver nenhum contêiner inicialização (init container) configurado, todos os contêineres nesse Pod serão contêineres de aplicação.
```

Comment:
Minor: "contêiner de inicialização" like line 13 and 17. 

I think the "(init container)" is no longer needed because you already made the reference in line 13. Think about it.

Reply:
Agreed!